x/nftmarket/keeper/nft_listing.go

```diff
@@ -196,6 +247,8 @@ func (k Keeper) ListNft(ctx sdk.Context, msg *types.MsgListNft) error {
 	}
 
 	params := k.GetParamSet(ctx)
+	// todo: delete
+	params.BidTokens = append(params.BidTokens, "uguu")
```

Comment:
These test code will need to be removed.

Reply:
I fixed